app/assets/stylesheets/public/_toc.scss

```diff
@@ -2,6 +2,8 @@
   box-sizing: border-box;
 
   @media (min-width: $screen-lg) {
+    background-color: #fff;
+    border-left: #ddd 1px solid;
```

Comment:
In case of content overlap, the white background will cover up what's behind the sidebar

Fix left border that got accidentally left out.